src/core/core.controller.js

```diff
@@ -958,14 +984,18 @@ class Chart {
   unbindEvents() {
     const me = this;
     const listeners = me._listeners;
-    if (!listeners) {
-      return;
-    }
 
     me._listeners = {};
     each(listeners, (listener, type) => {
```

Comment:
`listeners` only used once, so might as well use `me._listeners` instead and remove the variable. Need to move the `me._listeners = {};` after the loop too.